### ISSUE TITLE
--regions: a BED file containing regions to be used only in genome-scan

### ISSUE THREAD

**Issue by minw2828** [bcgsc/straglr#47]
Hello,

Thank you for developing the tool.

If I am expecting a repeat may or may not start with the `--regions regions.bed`, but will extend beyond the genomic regions that are defined within `--regions regions.bed`,  would `straglr` be able to find it? 

Many thanks,
Min

**Comment by readmanchiu** [comment]
Yes, you can define your region to whatever you like. I would not recommend centromeric or regions densely populated with tandem repeats. Also by specifying `--regions` instructs Straglr to look for repeat expansions relative to the reference genome, it would not report sizing of all tandem repeats within the region if they are not expanded.
